#DoCoMo ScenarioDialogue API Response message
Header header
string projectKey
string appName
string appKey
string clientVer
string dialogMode
string language
string userId
string commandId
string commandName
string slotName
string slotValue
string valueType
string contentSource
string contentType
string contentEncode
string contentValue
string projectSpecific
string utteranceText
string utteranceRevised
string utteranceWord
string taskIdList
string wordsValue
string wordsType
string serverSendTime

